### PR TITLE
Create Howard Hughes Medical Institute 2023.csl

### PR DESCRIPTION
**Pull request by michavissar**
This is a house citation style for the Howard Hughes Medical Institute created in 2023.

### CODE REVIEW

File: Howard Hughes Medical Institute 2023.csl [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Howard Hughes Medical Institute 2023</title>
```
Comment (bwiernik) [inline Howard Hughes Medical Institute 2023.csl:5]:
```suggestion
    <title>Howard Hughes Medical Institute</title>
```

**Review by bwiernik — DISMISSED** [review]
Not sure why the automated validation report isn't showing. Sorry about that. 

Left several comments to fix. Also, please be sure your style validates by running it through 

https://validator.citationstyles.org/

**Comment by adam3smith** [comment]
Also, see https://github.com/citation-style-language/styles/blob/master/STYLE_REQUIREMENTS.md which includes most of the points bwiernik mentions and fix the rights section. It needs to be _exactly_ as listed there; your text differs slightly.

Also, don't use validator.nu -- use our custom validator at https://validator.citationstyles.org/

File: Howard Hughes Medical Institute 2023.csl
[the remark this answers is not in the document]
Comment (michavissar) [reply Howard Hughes Medical Institute 2023.csl:1]: will do - thank you!

File: Howard Hughes Medical Institute 2023.csl [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+<citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout delimiter="," vertical-align="sup">
+      <text variable="citation-number"/>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true" entry-spacing="0" line-spacing="1">
+    <text variable="citation-number" suffix=". "/>
```
[the remark this answers is not in the document]
Comment (michavissar) [reply Howard Hughes Medical Institute 2023.csl:115]: Thanks, and sorry for my confusing (I'm a complete n00b and this is a one-off project). Initially, this comment seemed to be referring to lines 112-115, but now I'm seeing a much larger block of code. Can you clarify which lines need to move?

[the remark this answers is not in the document]
Comment (bwiernik) [reply Howard Hughes Medical Institute 2023.csl:115]: Line 115 — the citation-number text line.

**Comment by michavissar** [comment]
Thanks so much for all the help @adam3smith and @bwiernik . Greatly appreciated!

**Comment by adam3smith** [comment]
(Please updated this PR instead of opening a new one)

**Comment by michavissar** [comment]
Oops, I moved too quickly. I've close the new one now. I made a number of changes beyond those listed here. Can you explain how I replace the old code with the new one for review?

**Comment by adam3smith** [comment]
> Can you explain how I replace the old code with the new one for review?

Click on "Files Changed", then on the three-dots menu on the right and edit file, delete the old code and replace with the new one. You can have github show you a preview of your changes before committing.

**Comment by adam3smith** [comment]
The failing check here is some technical glitch, don't worry about it.

**Comment by adam3smith** [comment]
@michavissar -- please confirm that the preview above looks right & we can merge

**Comment by michavissar** [comment]
> @michavissar -- please confirm that the preview above looks right & we can merge

Everything looks good - thank you!

**Pull request merged**